lib/promise-polyfill.js

```diff
@@ -373,16 +373,18 @@ Promise.race = function(arr) {
 };
 
 // Use polyfill for setImmediate for performance gains
-Promise._immediateFn =
+// @ts-ignore
```

Comment:
Please update the `Patches for use in QUnit:` header to document this local patch, to make sure we remember it in the future :)